src/jsMain/kotlin/net/kyori/adventure/webui/js/Main.kt

```diff
@@ -533,7 +642,7 @@ private fun parse() {
                     if (line == "") "\u200B" else line
                 }
 
-            webSocket.send(Serializers.json.encodeToString(Call(combinedLines)))
+            webSocket.send(Serializers.json.encodeToString(Call(combinedLines) as Packet))
```

Comment:
Minor improvement, but an extension function on websocket that handles all this encoding/casting (is the casting even needed?) might be nice so we can just do `webSocket.send(packet)`

Reply:
Casting is needed for it to do "polywhatever" stuff, i.e. include the "type" field

(this is my first time using kotlinx serialization)